assets/src/blocks/TakeActionBoxout/TakeActionBoxoutFrontend.js

```diff
@@ -13,15 +11,17 @@ export const TakeActionBoxoutFrontend = ({
 }) => (
   <section
     className={`boxout ${className || ''}`}
-    {...stickyOnMobile && { id: 'action-card' }}
+    {...stickyOnMobile && {id: 'action-card'}}
   >
+    {/* eslint-disable-next-line jsx-a11y/anchor-has-content */}
     <a
       data-ga-category="Take Action Boxout"
       data-ga-action="Image"
       data-ga-label="n/a"
       className="cover-card-overlay"
       href={link}
-      {...newTab && link && { target: "_blank" }}
+      target={(newTab && link) ? '_blank' : ''}
```

Comment:
Same comment for this file as earlier, we should set `rel="noreferrer"` only when `target` is `_blank`